tests/e2e/device/e2e_device_commands.c

```diff
@@ -780,6 +785,150 @@ static uint32_t prvE2ETestVerifyDesiredPropertiesCommandExecute( E2E_TEST_COMMAN
 }
 /*-----------------------------------------------------------*/
 
+/**
+ * Execute Writeable property response command
+ *
+ * e.g of command issued from service:
+ *   {"method":"get_writable_properties_repsonse","desired_property_key":"temp","desired_property_value":"7664262703"}
+ *
+ **/
+static uint32_t prvE2ETestWriteablePropertyResponseCommandExecute( E2E_TEST_COMMAND_HANDLE xCMD,
+                                                                   AzureIoTHubClient_t * pxAzureIoTHubClient )
+{
+    uint32_t ulStatus;
+    uint16_t usTelemetryPacketID;
+    uint32_t ulRequestId;
+    az_span xJsonSpan = az_span_create( ( uint8_t * ) xCMD->pulReceivedData, xCMD->ulReceivedDataLength );
+    az_span xKey = az_span_create( ucScratchBuffer, e2etestMESSAGE_BUFFER_SIZE / 3 );
+    az_span xValue = az_span_create( ucScratchBuffer + e2etestMESSAGE_BUFFER_SIZE / 3, e2etestMESSAGE_BUFFER_SIZE / 3 );
+    az_span xValueReceived = az_span_create( ucScratchBuffer + 2 * e2etestMESSAGE_BUFFER_SIZE / 3, e2etestMESSAGE_BUFFER_SIZE / 3 );
+    AzureIoTJSONWriter_t xJSONWriter;
+
+    ulStatus = prvGetValueForKey( xJsonSpan, e2etestDESIRED_PROPERTY_KEY, &xKey );
+    RETURN_IF_FAILED( ulStatus, "failed to find desired_property_key!" );

```

Comment:
nit: here and 811 for consistency
```suggestion
    RETURN_IF_FAILED( ulStatus, "Failed to find desired_property_key!" );

```